Lib/concurrent/futures/process.py

```diff
@@ -855,3 +868,65 @@ def shutdown(self, wait=True, *, cancel_futures=False):
         self._executor_manager_thread_wakeup = None
 
     shutdown.__doc__ = _base.Executor.shutdown.__doc__
+
+    def _terminate_or_kill_workers(self, operation: _TerminateOrKillOperation):
```

Comment:
I don't think we add typing in general and we leave it to typeshed. Otherwise I would add a return annotation as well.